lib/msal-common/src/logger/Logger.ts

```diff
@@ -62,10 +62,16 @@ export class Logger {
         const defaultLoggerCallback = () => {
             return;
         };
-        this.localCallback = loggerOptions.loggerCallback || defaultLoggerCallback;
-        this.piiLoggingEnabled = loggerOptions.piiLoggingEnabled || false;
-        this.level = typeof(loggerOptions.logLevel) === "number" ? loggerOptions.logLevel : LogLevel.Info;
-        this.correlationId = loggerOptions.correlationId || Constants.EMPTY_STRING;
+        const defaultLoggerOptions = {};
+        if(!loggerOptions)
```

Comment:
this is added for coherence with defaultLoggerCallback, can be removed based on the feedback as it doesnt impact the working/tests.

Reply:
Logger options is a required parameter so this should never be hit